README.md

```diff
@@ -1,110 +1,148 @@
-Heliclockter
-=======
+# Heliclockter
 
-`heliclockter` is a robust way of dealing with datetimes and timestamps in python. It is statically
-type checkable as well as runtime enforceable and integrates with [pydantic][pydantic].
+[![PyPI](https://img.shields.io/pypi/v/heliclockter)](https://pypi.org/project/heliclockter/)
+[![License](https://img.shields.io/github/license/channable/heliclockter)](https://github.com/channable/heliclockter/blob/master/LICENSE)
+[![Python Versions](https://img.shields.io/pypi/pyversions/heliclockter)](https://pypi.org/project/heliclockter/)
 
-The library exposes 3 classes:
+**Timezone-aware datetimes for Python that just work.**
 
-- `datetime_tz`, a datetime ensured to be timezone-aware.
-- `datetime_local`, a datetime ensured to be timezone-aware in the local timezone.
-- `datetime_utc`, a datetime ensured to be timezone-aware in the UTC+0 timezone.
+`heliclockter` is a timezone-aware datetime library that ensures your timestamps are always timezone-aware. It's statically type checkable and runtime enforceable.
 
-as well as various utilities to instantiate, mutate and serialize those classes.
+## Installation
 
-See our [announcement post][announcement] for a more background on why we wrote `heliclockter`.
-
-[pydantic]: https://github.com/pydantic/pydantic
-[announcement]: https://www.channable.com/tech/heliclockter-timezone-aware-datetimes-in-python
-
-Examples
--------
+```bash
+pip install heliclockter
+```
 
-Say you want to create a timestamp of the current time in the UTC+0 timezone.
+## Quick Start
 
 ```python
-from heliclockter import datetime_utc
+from heliclockter import datetime_utc, datetime_local, datetime_tz
 
-now = datetime_utc.now()
+# UTC datetime
+utc_now = datetime_utc.now()
 # datetime_utc(2022, 11, 4, 15, 28, 10, 478176, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
-```
 
-Or imagine you want to create a timestamp 2 hours in the future from now:
+# Local timezone datetime  
+local_now = datetime_local.now()
 
-```python
-from heliclockter import datetime_utc
+# Any timezone datetime
+paris_tz = datetime_tz.now("Europe/Paris")
```

Comment:
This won't work with a positional argument. You are now passing `"Europe/Paris"` to the `year` argument of the constructor. Also the tzinfo argument cannot be a string, it must be a `tzinfo` object, specifically we use `ZoneInfo`